app/src/frontend/building/edit-history/field-edit-summary.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface FieldEditSummaryProps {
+    title: string;
+    value: any;
+    oldValue: any;
+}
+
+const FieldEditSummary: React.FunctionComponent<FieldEditSummaryProps> = props => (
+    <>
+        {props.title}:&nbsp;
+        <code title="Value before edit" className='edit-history-diff old'>{props.oldValue}</code>
+        &nbsp;
+        <code title="Value after edit" className='edit-history-diff new'>{props.value}</code>
```

Comment:
I think the styling for the old/new value is sufficient to convey the edit history with colours. Not 100% sure, though, if using the `title` field with descriptions (`Value before edit` / `Value after edit`) is sufficient for accessibility